### PR TITLE
1050 utils tests

### LINKED ISSUE #1050: Util tests
there are a bunch of utils in the utils folder. They're very easy and quick to write tests for. They should have tests!

### PR DESCRIPTION
**Pull request by mark-prins**
Fixes #1050 

Adds a few tests for the utils

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -15,5 +15,6 @@ module.exports = {
     ...pathsToModuleNameMapper(compilerOptions.paths, {
       prefix: '<rootDir>/',
     }),
+    '.+\\.(gif)$': 'jest-transform-stub',
```
Comment (mark-prins) [inline jest.config.js:18]: the image import was causing an issue for jest, had to work around that.

File: packages/common/src/utils/arrays/ArrayUtils.test.ts [hunk 2]
```diff
@@ -0,0 +1,75 @@
+import { ArrayUtils } from './ArrayUtils';
+
+describe('ArrayUtils', () => {
+  it('is defined', () => {
+    expect(ArrayUtils.ifTheSameElseDefault).toBeDefined();
+    expect(ArrayUtils.getUnitQuantity).toBeDefined();
+    expect(ArrayUtils.getSum).toBeDefined();
+    expect(ArrayUtils.immutablePatch).toBeDefined();
```
Comment (clemens-msupply) [inline packages/common/src/utils/arrays/ArrayUtils.test.ts:8]: Is this testing something not already captured by the type system?

File: packages/common/src/utils/arrays/ArrayUtils.test.ts [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    expect(ArrayUtils.getSum(arr2, 'value')).toBe(1.5);
+  });
+
+  it('immutablePatch', () => {
+    const arr: { id: string; unit: string }[] = [
+      { id: '1', unit: 'bottle' },
+      { id: '2', unit: 'bottle' },
+      { id: '3', unit: 'tablet' },
+    ];
+    const newArray = ArrayUtils.immutablePatch(arr, {
```
Comment (clemens-msupply) [inline packages/common/src/utils/arrays/ArrayUtils.test.ts:66]: why is it called immutablePatch?

File: packages/common/src/utils/arrays/SortUtils.test.ts [hunk 4]
```diff
@@ -0,0 +1,40 @@
+import { SortUtils } from './SortUtils';
+
+describe('SortUtils', () => {
+  it('is defined', () => {
+    expect(SortUtils.byExpiryAsc).toBeDefined();
+    expect(SortUtils.byExpiryDesc).toBeDefined();
+    expect(SortUtils.getColumnSorter).toBeDefined();
+    expect(SortUtils.getDataSorter).toBeDefined();
```
Comment (clemens-msupply) [inline packages/common/src/utils/arrays/SortUtils.test.ts:8]: as previous comment...

**Review by clemens-msupply — APPROVED** [review]

File: packages/common/src/utils/arrays/ArrayUtils.test.ts (hunk 2 above)
Comment (mark-prins) [reply packages/common/src/utils/arrays/ArrayUtils.test.ts:8]: yeah it is, and that will be picked up in the pre-commit hook. tbh, I was being a little lazy and copied the existing pattern here, though these tests don't actually add anything very helpful. tests for tests sake.. and I think I'll just remove them

File: packages/common/src/utils/arrays/ArrayUtils.test.ts (hunk 3 above)
Comment (mark-prins) [reply packages/common/src/utils/arrays/ArrayUtils.test.ts:66]: as I remember it.. just because the original array is unchanged

**Pull request merged**